MakefileConstants.mk

```diff
@@ -5,7 +5,10 @@
 # Main parameters
 FEATURE_NAME ?= provision-account
 TEAM_NAME ?= cloudzero
-SRC_FILES = $(shell find . -name "*.yaml" -a \! -name "packaged*.yaml" -a \! -path "*.aws-sam*")
+BUCKET = cz-$(FEATURE_NAME)
```

Comment:
I take it this bucket is created in the reactor account?

Reply:
No ... this is the deployment bucket for these templates ... it's in prime like our other publicly downloadable resources.